.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,6 +13,7 @@
 - [ ] No unnecessary changes are introduced.
 - [ ] Description motivates each change.
 - [ ] Avoids breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
+- [ ] Backport labels are set and Changes(s) will backported to all supported release lines [release policy](https://ddtrace.readthedocs.io/en/latest/versioning.html).
```

Comment:
The wording of this checklist item and the link to the version policy feels a bit weird. Feel free to suggest alternatives.